prawcore/requestor.py

```diff
@@ -2,16 +2,13 @@
 
 from __future__ import annotations
 
-from typing import TYPE_CHECKING, Any
+from typing import Any
 
-import requests
+import niquests
 
 from .const import TIMEOUT
 from .exceptions import InvalidInvocation, RequestException
 
-if TYPE_CHECKING:
-    from requests.models import Response, Session
-
```

Comment:
Can we add this block back to keep consistent with the other modules?

Reply:
Done.
FWIW those are already accessible and loaded in memory from the root import. This line is redundant.